datajoint/admin.py

```diff
@@ -14,6 +15,10 @@ def set_password(new_password=None, connection=None):   # pragma: no cover
     connection.query("SET PASSWORD = PASSWORD('%s')" % new_password)
     print('Password updated.')
 
+    if re.match('([Yy]|[Yy][Ee][Ss])', input('update dj_local_conf.json? ')):
```

Comment:
Cool - would you mind adding a keyword argument `prompt=True` or something like that so if set to `False`, it will not prompt to update? I'm thinking of the case we want to use this in a script or in batch where a prompt will be annoying :)

Reply:
Or maybe `update=None` and if it is set to `True` or `False`, it will use that as the response without prompting but if `None` (default), it prompts to udpate?

Reply:
all set